components/ui/TypedWord.vue

```diff
@@ -0,0 +1,91 @@
+<template>
+  <div class="typed-text__container" :class="{ 'typed-text__cursor': showCursor }">
+    <span class="typed-text__content">{{typeValue}}</span>
+  </div>
+</template>
+
+<script>
+export default {
+  data: () => {
+    return {
+      typeValue: "",
+      typeStatus: false,
+      typeArray: ["Qiskit", "Python", "Quantum"],
+      typingSpeed: 200,
+      erasingSpeed: 100,
+      newTextDelay: 3000,
+      eraseTextDelay: 5000,
+      typeArrayIndex: 0,
+      charIndex: 0,
+      showCursor: true,
```

Comment:
We should identify the public API of the component (those values that customize the behaviour of the component) and convert them into component properties so it is the consumer of the component who's in charge of customizing the behaviour to their needs.

The way it is right now implies a strong coupling with the learn page.